.github/workflows/ci-test-ginkgo.yml

```diff
@@ -78,17 +78,7 @@ jobs:
             /tmp/kubearmor/
             /tmp/kubearmor.*
 
-      - name: Measure code coverage
-        if: ${{ always() }}
-        run: |
-          go install github.com/modocache/gover@latest
-          gover
-          go tool cover -func=gover.coverprofile
-        working-directory: KubeArmor
-        env:
-          GOPATH: /home/runner/go
-
-      - uses: codecov/codecov-action@v3
-        if: ${{ always() }}
+      - name: Upload code coverage
+        uses: actions/upload-artifact@v3 
         with:
           files: ./KubeArmor/gover.coverprofile
```

Comment:
We are trying to upload ./KubeArmor/gover.coverprofile but you removed the code generate that 😅, ig it's failing that;s why